asv_bench/benchmarks/benchmarks.py

```diff
@@ -149,19 +149,24 @@ def time_merge(self, shapes, how, sort):
 
 
 class TimeConcat:
```

Comment:
`ConcatDataFrames` is removed in favour of this benchmark with some new parameters (https://github.com/modin-project/modin/issues/5045 is blocker for using `ConcatDataFrames`).